diff -urN fceux-2.2.1/src/drivers/sdl/sdl.cpp fceux-2.2.1-patched/src/drivers/sdl/sdl.cpp
--- fceux-2.2.1/src/drivers/sdl/sdl.cpp	2013-03-03 06:46:34.000000000 +1100
+++ fceux-2.2.1-patched/src/drivers/sdl/sdl.cpp	2013-03-12 18:21:51.530198502 +1100
@@ -934,7 +934,7 @@
 	if(gtkIsStarted == true && noGui == 0)
 	{
 		GtkWidget* d;
-		d = gtk_message_dialog_new(GTK_WINDOW(MainWindow), GTK_DIALOG_MODAL,        GTK_MESSAGE_ERROR, GTK_BUTTONS_OK, errormsg);
+		d = gtk_message_dialog_new(GTK_WINDOW(MainWindow), GTK_DIALOG_MODAL,        GTK_MESSAGE_ERROR, GTK_BUTTONS_OK, "%s", errormsg);
 		gtk_dialog_run(GTK_DIALOG(d));
 		gtk_widget_destroy(d);
 	}
